extensions/wikia/FounderEmails/events/FounderEmailsEditEvent.class.php

```diff
@@ -32,7 +34,109 @@ public function __construct( Array $data = array() ) {
 		$this->setData( $data );
 	}
 
-	public function enabled ( $wgCityId, User $user ) {
```

Comment:
This ```enabled``` function was just moved further down.  The diff ended up a little goofy ...